### PR TITLE
Add missing documentation to Task model

### PR DESCRIPTION
**Pull request by shivamjagdishsingh**
### Task Description
- Add documentation to Task model.

### CODE REVIEW

**Review by ruebenramirez — COMMENTED** [review]
Nice add!

**Review by ruebenramirez — APPROVED** [review]

**Comment by shivamjagdishsingh** [comment]
@ruebenramirez 
Thank you for the approval. 
But, I am still unable to merge the PR. I says need approval from reviewer with write access. 
Can something be done about it?

**Comment by joaomdmoura** [comment]
Great  improvement, let's go!

**Pull request merged**
